core/src/main/kotlin/com/linecorp/kotlinjdsl/querydsl/QueryDslImpl.kt

```diff
@@ -129,8 +129,8 @@ open class QueryDslImpl<T>(
         lazyJoins().add(FetchJoinSpec(left = left, right = right, path = relation.path, joinType = joinType))
     }
 
-    override fun where(predicate: PredicateSpec) {
-        lazyWheres().add(predicate)
+    override fun where(vararg predicates: PredicateSpec?) {
+        for (p in predicates) p?.let { lazyWheres().add(it) }
```

Comment:
-- korean
where 메소드에 vararg로 predicates를 받는 것은 사용자에게 혼란을 발생시킬 수 있을 것 같습니다. predicates가 and로 묶일지 or로 묶일지 보이지 않기 때문입니다.

그래서 where 메소드는 `predicate: PredicateSpec?`로 단일 Predicate만 받고, `whereAnd(vararg perdicates: PredicateSpec?)` `whereAnd(vararg perdicates: PredicateSpec?)` 과 같은 and로 묶일지 or로 묶일지 나타내는 메소드를 추가로 작성하면 사용할 때에도 문제 없을 것으로 보입니다.

또한 vararg로만 만드는 것이 아니라 List로 만드는 것도 좋을 것 같습니다. 사용자가 Predicate를 List 변수로 만들어 사용할 수 있기 떄문입니다.

-- english
Receiving predicates with vararg in the where method may cause confusion for the user. Because user can't know if the predicates are combined with `and` or `or`.

It seems that there will be no additional problem if you take a single predicate like `where(vararg predicate: PredicateSpec?)`, and It would be good to add methods to indicate whether it will be combined with `and` or `or`, such as `whereAnd(vararg perdicates: PredicateSpec?)` `whereAnd(vararg perdicates: PredicateSpec?)`.

Also, I think it would be good to make to receive list parameter, not just vararg. This is because the user can use Predicate as a List variable.